common/src/Microsoft.Azure.IIoT.Hub.Module.Framework/src/Hosting/MethodRouter.cs

```diff
@@ -99,7 +99,7 @@ public sealed class MethodRouter : IMethodRouter, IMethodHandler {
         /// <param name="target"></param>
         private void AddToCallTable(object target) {
             var versions = target.GetType().GetCustomAttributes<VersionAttribute>(true)
-                .Select(v => "_v" + v.Value)
```

Comment:
@monikavar, can we please add in your documentation changes that both versioned and non-versioned DM names are supported, but we strongly recommend to use versioned names.

Reply:
I have already added this in the PR.

Reply:
Thank you.